app/validators/csv/facilities_validator.rb

```diff
@@ -29,7 +29,9 @@ def at_least_one_facility
   end
 
   def duplicate_rows
-    fields = facilities.map { |facility| facility.slice(:organization_name, :facility_group_name, :name) }
+    fields = facilities
+      .map { |facility| facility.slice(:organization_name, :facility_group_name, :name) }
+      .map { |facility| facility[:name] = facility[:name].gsub(/\s+/, "") }
```

Comment:
Wouldn't this `map` return just a list of facility names? The other attributes will not be uniqued by

```
> [{a: 1}, {b: 2}].map { |a| a[:name] = "Some name" }
=> ["Some name", "Some name"]
```
```

Reply:
d'oh